addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -455,7 +455,48 @@ export class TablePlugin extends Plugin {
         return true;
     }
 
+    hanldeFirefoxSelection(ev = null) {
+        const selection = this.document.getSelection();
+        const isFirefox = navigator.userAgent.toLowerCase().indexOf("firefox") > -1;
+        if (isFirefox) {
+            if (selection.rangeCount > 1) {
+                // In Firefox, selecting multiple cells within a table using the mouse can create multiple ranges.
+                // This behavior can cause the original selection (where the selection started) to be lost.
+                // To address this, we reset the selection to the _latestComputedSelection, ensuring that
+                // even when multiple ranges are selected, the original selection remains accessible.
```

Comment:
This comment does not seem to match what the code does.

Reply:
Fixed it thanks, I changed the fix and I missed the comment